eliot/_util.py

```diff
@@ -37,3 +39,22 @@ def saferepr(o):
     except:
         # Not much we can do about this...
         return "eliot: unknown, unicode() raised exception"
+
+
+def load_module(name, original_module):
+    """
+    Load a copy of a module, distinct from what you'd get if you imported
+    it directly.
+
+    @param str name: The name of the new module.
+    @param original_module: The original module we're recreating.
+
+    @return: A new, distinct module.
+    """
+    module = ModuleType(name)
+    path = original_module.__file__
+    if path.endswith(".pyc") or path.endswith(".pyo"):
+        path = path[:-1]
```

Comment:
:question: I don't think you can assume that the `.py` file is always alongside the `.pyc` or `.pyo`. I seem to remember that Ubuntu puts them in a separate directory (in Python3 only??).
And  there may be cases where the py files are not distributed at all.
But reading down I see that this is only used for patching the traceback module....and that it's a refactoring of code that already existed and had already been reviewed. So fine.